server/app/controllers/receive.py

```diff
@@ -0,0 +1,118 @@
+import abc
+import datetime
+import os.path
+import uuid
+
+from flask import request, abort
+from webargs import fields
+from webargs.flaskparser import use_args
+from werkzeug.utils import secure_filename
+
+from app import app
+from app.authorize_station import authorize_station
+from app.utils import make_thumbnail, first
+from app.repository import Observation, ObservationFile, ObservationFileId, ObservationId, Repository, SatelliteId, StationId
+from abc import abstractmethod
+
+import sys
+if sys.version_info >= (3, 8):
+    from typing import TypedDict
+else:
+    from typing_extensions import TypedDict
+
+class WebFileLike(abc.ABC):
```

Comment:
This class requires a comment: what is it for? what's ABC?

Reply:
abc module is Python-way for define abstract class, methods, collections.

Reply:
I noticed that I don't use this class. I fix it.